### PR TITLE
Change to flutter pub

### PR DESCRIPTION
**Pull request by mit-mit**
Related to flutter/flutter#33041

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,10 +132,10 @@ and then you may use the `webdev` command directly from your terminal.
 2. Update packages.
 
     ```console
-    $ flutter packages upgrade
+    $ flutter pub upgrade
```
Comment (yjbanov) [inline README.md]: I believe it's `flutter pub pub upgrade`

File: README.md [hunk 2]
```diff
@@ -132,10 +132,10 @@ and then you may use the `webdev` command directly from your terminal.
...
     ! flutter_web 0.0.0 from path ../../flutter_web
     ! flutter_web_ui 0.0.0 from path ../../flutter_web_ui
-    Running "flutter packages upgrade" in hello_world...                5.0s
+    Running "flutter pub upgrade" in hello_world...                5.0s
```
Comment (yjbanov) [inline README.md]: ditto

File: README.md (hunk 1 above)
Comment (mit-mit) [reply README.md]: No longer needed! See https://github.com/flutter/flutter/pull/30115

**Comment by yjbanov** [comment]
Cool! I'll pull this change in (we do not accepts PRs directly yet)

**Pull request closed without merging**